wdio/specs/homepage.js

```diff
@@ -117,4 +117,25 @@ describe('Municipality homepage', () => {
     const target = browser.getAttribute('.item.action.homepage-teaser:nth-child(4) .content .header a', 'target');
     assert(target == '_blank');
   });
+
+  it('should show QAs only for the chosen user type "residents" and the hebrew language', () => {
+    browser.url('/municipality-1/node/1?user_type=residents&language=he');
+    browser.waitForVisible('div=כמה סייעות יש בגן ילדים?');
+    browser.waitForVisible('div=באילו שעות מפנים את הזבל?');
+  });
+
+  it('should not show QAs for "businesses" when "residents" are the chosen user type ', () => {
+    assert(!browser.isVisible('div= האם גני ילדים זכאים לפטור מארנונה?'));
```

Comment:
I've checked all municipalities, there is only one question for each user type +  1 question for both of them = 3 
There is nothing else to check, maybe we should create at least 2 questions in the migration process just to be sure.